### PR TITLE
Bp/aga ftr improvement

### LINKED ISSUE #279: Improvement of AGA/FTR invoice
In the comment of the invoice that is for AGA/FTR it should have an indication if this report is replacing an old one. For example: Erstatter AR136293635. Same as on AGA/FTR print page in the last line.

Make only one button for sending both AGA and FTR type invoices to fakturaBank. And the old info on who sent the reports and when is to be migrated to the latest one.

### PR DESCRIPTION
**Pull request by Baxxx**
Closes #279 
Rebase and merge after EHF merging, last commit is relevant

### CODE REVIEW

File: modules/fakturabank/model/fakturabank.class.php [hunk 1]
```diff
@@ -2239,8 +2240,8 @@ public function save_invoice_export_data($data) {
           }
           unset($_SESSION['altinn_invoice_reference']);
           unset($_SESSION['altinn_invoice_sending']);
-          unset($_SESSION['altinn_invoice_type']);
           $_lib['storage']->store_record(array('data' => $dataH, 'table' => 'altinnReport4', 'debug' => false));
+          $_SESSION['oauth_invoice_error'][] = $_SESSION['altinn_invoice_type'].": Success";
```
Comment (Baxxx) [inline modules/fakturabank/model/fakturabank.class.php:2260]: Unset `$_SESSION['altinn_invoice_type']`

**Comment by sls-lnluksa** [comment]
@stoivo 
This PR is related to EHF rework.
Deploy only when EHF branch is on live.

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

**Comment by Baxxx** [comment]
When token is expired, AGA is sent twice, and FTR isn't received.

**Comment by sls-lnluksa** [comment]
First round review: Nikola Lukovic (sls-lnluksa)
Final review with: @stoivo

@stoivo 
This PR is related to EHF rework.
Deploy only when EHF branch is on live.

**Pull request merged**